app/assets/stylesheets/main.css.scss

```diff
@@ -2251,3 +2251,11 @@ figure.fluidratio {
     }
   }
 }
+
+.selection-options {
+  @include media(tablet) {
+    margin-top: lines(0.5);
+    margin-bottom: em(3);
+    float: right;
+  }
+}
```

Comment:
I've avoided adding anything new to the `main.css.scss`, since it's an uncategorised code dumpster. Could you move this somewhere else? Maybe `stylesheets/partials/forms.css.scss` ?